pllug-presentation-system/pandoc-workflow/pandocrunner.cpp

```diff
@@ -0,0 +1,64 @@
+#include "pandocrunner.h"
+
+#include <QCoreApplication>
+#include <QDir>
+
+#include <QDebug>
+
+PandocRunner::PandocRunner(const QStringList &params, QObject* parent):
+    QProcess(parent)
+  , mParams(params)
```

Comment:
But we need some method, or associative container, for example, map, to convert enum value to real pandoc parameter.